pkg/userdata/coreos/provider.go

```diff
@@ -311,6 +311,15 @@ storage:
           [Service]
           Environment=DOCKER_OPTS=--storage-driver=overlay2
 
+    - path: /etc/systemd/system/kubelet.service.d/11-cgroups.conf
```

Comment:
If we add this for all OS, can you please just change the `kubelet.service` unitfile instead of creating a drop-in?

Reply:
Sounds good. The original fix #476 has been a copy of the solution of https://github.com/kubernetes/kubernetes/issues/56850 . As far as my checks have shown there are no global default settings of these two switches (all commented out) as well no settings in the `kubelet.service` right now.